defopt.py

```diff
@@ -53,17 +55,19 @@ def run(*funcs, **kwargs):
     :param parsers: Dictionary mapping types to parsers to use for parsing
         function arguments.
     :type parsers: Dict[type, Callable[[str], type]]
-    :param short: Dictionary mapping parameter names to letters to use as
-        alternative short flags.
+    :param short: Dictionary mapping parameter names, after conversion of
+        underscores to dashes, to letters, to use as alternative short flags.
```

Comment:
Nice catch. I find this hard to parse with the commas - could you instead put the added bit in parentheses?